SUMMARY.md

```diff
@@ -8,8 +8,8 @@
   - [Agents as microservices](./docs/design-options/Microservices.md)
 - [Agents registry](./docs/agent-registry/Agent-Registry.md)
 - [Agents communication](./docs/agents-communication/Agents-Communication.md)
-  - [Request-driven](./docs/agents-communication/Request-Driven.md)
-  - [Event-driven]()
+  - [Request-based](./docs/agents-communication/Request-Driven.md)
```

Comment:
Hey @fedeoliv, don't we need to use the same name as the file name for the mdbooks?